getting-started/macos/macos-basic-setup.md

```diff
@@ -0,0 +1,80 @@
+---
+ID:
+post_title: Basic Roots development environment for macOS
+author: Michael W. Delaney
+post_excerpt: ""
+layout: doc
+permalink: >
+
+published: false
+post_date:
+---
+
+Basic macOS 10.13+ development environment for Roots Trellis, Bedrock, and Sage
+
+## Requirements
```

Comment:
How about adding instructions to run Software Update, at least for any system updates, as a first step?

Reply:
That steps dangerously close to general IT support I think.